scripts/environments/languageserver/Manifest.toml

```diff
@@ -45,38 +81,76 @@ path = "../../packages/JSONRPC"
 uuid = "b9b8584e-8fd3-41f9-ad0c-7255d428e418"
 version = "1.3.4-DEV"
 
+[[JuliaFormatter]]
+deps = ["CSTParser", "CommonMark", "DataStructures", "Pkg", "Tokenize"]
+path = "../../packages/JuliaFormatter"
+uuid = "98e50ef6-434e-11e9-1051-2b60c6c9e899"
+version = "0.15.2"
+
 [[LanguageServer]]
-deps = ["CSTParser", "DocumentFormat", "JSON", "JSONRPC", "Markdown", "Pkg", "REPL", "StaticLint", "SymbolServer", "Tokenize", "URIParser", "UUIDs"]
+deps = ["CSTParser", "JSON", "JSONRPC", "JuliaFormatter", "Markdown", "Pkg", "REPL", "StaticLint", "SymbolServer", "Tokenize", "URIParser", "UUIDs"]
 path = "../../packages/LanguageServer"
 uuid = "2b0e0bc5-e4fd-59b4-8912-456d1b03d8d7"
 version = "4.1.1-DEV"
 
+[[LibCURL]]
+deps = ["LibCURL_jll", "MozillaCACerts_jll"]
+uuid = "b27032c2-a3e7-50c8-80cd-2d36dbcbfd21"
+
+[[LibCURL_jll]]
+deps = ["Artifacts", "LibSSH2_jll", "Libdl", "MbedTLS_jll", "Zlib_jll", "nghttp2_jll"]
+uuid = "deac9b47-8bc7-5906-a0fe-35ac56dc84c0"
```

Comment:
I think this manifest file was created with a recent Julia version, right? I don't think this will run on Julia 1.0, or for that matter on any pre artifact version of Julia. The auto-update script for these environments always uses Julia 1.3.1 so that these files are in a backwards compatible format, which we probably need to do here as well.